scripts/tax-id-analysis/checkDuplicateTaxIds.ts

```diff
@@ -0,0 +1,115 @@
+/**
```

Comment:
This and the similarly named .py file are almost identical. Initially written as a python script for ease of data manipulation. Opted for TS so that I could utilize `AWSEncryptionDecryptionFactory`.